install/operator/pkg/syndesis/backup/backup.go

```diff
@@ -542,6 +542,9 @@ func (b *Backup) backupDatabase() error {
 	if err != nil {
 		return err
 	}
+	if err = sc.ExternalDatabase(b.context, b.client, b.syndesis); err != nil {
```

Comment:
@phantomjinx  Did it fail before because the errors havn't been caught?

Reply:
It failed for a couple of reasons.
The ExternalDatabase was not being set in the job pod environment so it was trying to backup syndesis-db (default), which was of course not there. Secondly, the pod was being 'successful' as I had not added checks on the backup dump file being empty, ie. erroneous success status. Both addressed by this and fixed.